iceoryx2-bb/elementary/src/placement_default.rs

```diff
@@ -10,177 +10,5 @@
-Impl!(IoxAtomicU32);
-Impl!(IoxAtomicU64);
-Impl!(IoxAtomicI8);
-Impl!(IoxAtomicI16);
-Impl!(IoxAtomicI32);
-Impl!(IoxAtomicI64);
-Impl!(IoxAtomicIsize);
-Impl!(IoxAtomicUsize);
-
-impl<T: PlacementDefault> PlacementDefault for [T] {
-    unsafe fn placement_default(ptr: *mut Self) {
-        let ptr = ptr as *mut [MaybeUninit<T>];
-        for i in (*ptr).iter_mut() {
-            PlacementDefault::placement_default(i.as_mut_ptr());
-        }
-    }
-}
-
-impl<T: PlacementDefault, const CAPACITY: usize> PlacementDefault for [T; CAPACITY] {
-    unsafe fn placement_default(ptr: *mut Self) {
-        for i in 0..CAPACITY {
-            PlacementDefault::placement_default((ptr as *mut T).add(i))
-        }
-    }
-}
-
-impl<T1: PlacementDefault> PlacementDefault for (T1,) {
-    unsafe fn placement_default(ptr: *mut Self) {
-        let ptr = core::ptr::addr_of_mut!((*ptr).0);
-        PlacementDefault::placement_default(ptr)
-    }
-}
-
-impl<T1: PlacementDefault, T2: PlacementDefault> PlacementDefault for (T1, T2) {
-    unsafe fn placement_default(ptr: *mut Self) {
-        let elem = core::ptr::addr_of_mut!((*ptr).0);
-        PlacementDefault::placement_default(elem);
-
-        let elem = core::ptr::addr_of_mut!((*ptr).1);
-        PlacementDefault::placement_default(elem)
-    }
-}
-
-impl<T1: PlacementDefault, T2: PlacementDefault, T3: PlacementDefault> PlacementDefault
-    for (T1, T2, T3)
-{
-    unsafe fn placement_default(ptr: *mut Self) {
-        let elem = core::ptr::addr_of_mut!((*ptr).0);
-        PlacementDefault::placement_default(elem);
-
-        let elem = core::ptr::addr_of_mut!((*ptr).1);
-        PlacementDefault::placement_default(elem);
-
-        let elem = core::ptr::addr_of_mut!((*ptr).2);
-        PlacementDefault::placement_default(elem)
-    }
-}
-
-impl<T1: PlacementDefault, T2: PlacementDefault, T3: PlacementDefault, T4: PlacementDefault>
-    PlacementDefault for (T1, T2, T3, T4)
-{
-    unsafe fn placement_default(ptr: *mut Self) {
-        let elem = core::ptr::addr_of_mut!((*ptr).0);
-        PlacementDefault::placement_default(elem);
-
-        let elem = core::ptr::addr_of_mut!((*ptr).1);
-        PlacementDefault::placement_default(elem);
-
-        let elem = core::ptr::addr_of_mut!((*ptr).2);
-        PlacementDefault::placement_default(elem);
-
-        let elem = core::ptr::addr_of_mut!((*ptr).3);
-        PlacementDefault::placement_default(elem)
-    }
-}
-
-impl<T> PlacementDefault for Option<T> {
-    unsafe fn placement_default(ptr: *mut Self) {
-        ptr.write(Option::default())
-    }
-}
-
-impl<T: PlacementDefault> PlacementDefault for core::mem::MaybeUninit<T> {
-    unsafe fn placement_default(_ptr: *mut Self) {}
-}
-
-impl<T: Default> PlacementDefault for UnsafeCell<T> {
-    unsafe fn placement_default(ptr: *mut Self) {
-        ptr.write(UnsafeCell::default());
-    }
-}
+// re-export the PlacementDefault trait
+pub use iceoryx2_bb_elementary_traits::placement_default::*;
```

Comment:
Do we re-export it to here, and then also in lib.rs?

Reply:
removed